src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshaler.cs

```diff
@@ -596,6 +614,7 @@ public class ThrowingCleanUpNativeDataCustomMarshaler : ICustomMarshaler
 
         public static void Field_ParentIsStruct_ThrowsTypeLoadException()
         {
+            //  typeof(System.Runtime.InteropServices.MarshalDirectiveException): Struct 'StructWithCustomMarshalerField' requires marshalling that is not yet supported by this compiler.
```

Comment:
I think it should be fine to update this test to also accept MarshalDirectiveException. TypeLoadException is odd.

Reply:
I change this exception type in this PR. that was fairly trivial. I can move that change to separate PR if needed.

Reply:
Definitely a separate pull request. I somewhat doubt changing the exception thrown out of CoreCLR will be accepted, but one can always try.

Reply:
Remove code. Somebody has to rerun pipeline

Reply:
Is this comment still necessary?